ui/hooks/useTokenDisplayValue.js

```diff
@@ -43,7 +43,7 @@ export function useTokenDisplayValue(
       // and a token object has been provided
       token &&
       // and the provided token object contains a defined decimal value we need to calculate amount
-      token.decimals &&
+      token.decimals !== null &&
```

Comment:
```suggestion
      token.decimals != null &&
```

I wouldn't use strict equality here, since I don't think the token value should be calculated is `decimals` is `null` *or* `undefined`.

(Unless you have some evidence that it will always be `null`, but I'd expect it's more likely to be `undefined` anyway).

Perhaps an even better option would be `token.decimals >= 0` to protect against bad / negative inputs.

Reply:
the linter is failing, except if i disable the rule in this line